src/main/webapp/app/course/manage/course-management.service.ts

```diff
@@ -692,4 +695,12 @@ export class CourseManagementService {
         // Note: 0 is the default value in case the server returns something that does not make sense
         return this.http.get<number>(`${this.resourceUrl}/${courseId}/allowed-complaints?teamMode=${teamMode}`) ?? 0;
     }
+
+    setCourseOverviewBackground() {
+        this.courseOverviewSubject.next(true);
+    }
+
+    resetCourseOverviewBackground() {
+        this.courseOverviewSubject.next(false);
+    }
```

Comment:
Maybe `enable` and `disable` would be a better terminology here than set and reset